arkouda/numeric.py

```diff
@@ -1722,6 +1725,89 @@ def where(
     return create_pdarray(type_cast(str, repMsg))
 
 
```

Comment:
Before I review these, can you add unit tests to the PROTO_tests directory?

Reply:
Brainfade.  I had done that, and I think I lost it in the rebasing.  I'll reconstruct those tests later today.